app/posts/apply-for-teacher-training/2019-12-18-digital-accessibility-centre-report.md

```diff
@@ -0,0 +1,110 @@
+
+Automated accessibility testing would highlight this issue.
+
+### Unlabelled form fields (WCAG A)
+
+Page 15: When a training provider gives multiple conditions for an offer, the fields for those conditions don’t have associated labels.
+
+This was [fixed by removing custom IDs from there form fields](https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1015). The generated ID then matched the for attribute of the label, meaning that labels were correctly associated. ([Trello](https://trello.com/c/BWbCBEVj/693-dac-page-15-add-form-labels-to-provider-offer-page-conditions))
+
+Automated accessibility testing would highlight this issue.
+
+### Low colour contrast (WCAG AA)
+
+Page 17: The colour contrast of the declined (2.78:1) and offer (3.16:1) statuses shown to providers had insufficient contrast (should be 4.5:1).
+
+This was [fixed by changing our tag styles to use different colours](https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1011). ([Trello](https://trello.com/c/VjHpGzot/694-dac-page-17-fix-colour-contrast-of-declined-and-offer-status-on-applications-page))
+
+Many government services create tag styles using white text on the darker colours of the GOV.UK design system palette. Our new colours meet contrast ratios by using pastel colours, for example heavy red text on a light red background. So that other government services can benefit, and so we can stay inline with the design system, we've begun the process of [contributing these designs back upstream to GOV.UK Frontend](https://github.com/alphagov/govuk-frontend/pull/1711).
+
+Automated accessibility testing can usually highlight this type of issue, unless there are background images.
+
+### Error suggestion (WCAG AA)
+
+Page 20: When a candidate’s application was rejected without a reason a vague error message was shown.
+
+> The ‘can’t be blank’ error message presented on the ‘Reject application’ page when validation errors occur is not descriptive of what the error is and how to fix it.
+
+This was [fixed by changing the error message](https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1012) to "Enter feedback for the candidate". ([Trello](https://trello.com/c/7JcO0QxH/695-dac-page-20-add-error-text-for-blank-application-reject-reason))
+
+Automated accessibility testing would not have found this issue.
+
+### Reflow (WCAG AA)
+
+Page 23: On mobile devices links in the support page header were hidden and were not accessible.
+
+This has been fixed by showing a menu link which toggles hiding and showing of the links. It was [probably fixed when the header component was refactored](https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/982).
```

Comment:
This is because we had badly copied and pasted the Header component HTML from the design system.

I believe I fixed this in a separate PR and then refactored the Header component as it was an issue in lots of places.

Reply:
Can you dig out that PR?

Reply:
https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/970